pyracing/response_objects/iracing_data.py

```diff
@@ -28,3 +28,48 @@ def value_or_none(dict, field_name):
             return dict[field_name]
         else:
             return None
+
+
+class CarClass:
+    def __init__(self, dict):
+        self.rel_speed = dict['relspeed']  # Speed ranking to other classes
+        self.lower_name = dict['lowername']
+        self.custid = dict['custid']
+        self.class_full_name = dict['name']
+
+        # Inject into subclass CarsInClass?
+        self.carsinclass = dict['carsinclass']
+        self.class_id = dict['id']
+        self.class_shortname = dict['shortname']
+
+    class CarsInClass:
+        def __init__(self, dict):
+            self.car_name = dict['name']
+            self.car_id = dict['id']
+
+
+# Useful for personal_bests(). Need to know CarIDs to query.
+class CarsDriven:
+    def __init__(self, car_list):
+        self.list_of_cars = car_list
```

Comment:
This could also just be a list of that `Car` model i talked about in another comment.

Reply:
This one is different in that it returns only a list of car_id's. The other car object from carsinclass would actually have a dictionary of the name of the car and it's associated ID

Reply:
Ok, then lets rename it to `self.car_id_list` or something to make it more clear what it is and then that's fine